backends/openvino/tests/ops/base_openvino_op_test.py

```diff
@@ -0,0 +1,85 @@
+import unittest
+
+import executorch
+import torch
+from executorch.backends.openvino.partitioner import OpenvinoPartitioner
+from executorch.backends.openvino.preprocess import OpenvinoBackend
+from executorch.exir import EdgeProgramManager, to_edge_transform_and_lower
+from executorch.exir.backend.backend_details import CompileSpec
+
+from executorch.extension.pybindings.portable_lib import (  # @manual
+    _load_for_executorch_from_buffer,
```

Comment:
this pybind API is deprecated actually. Here's a new pybind API usage: https://colab.research.google.com/drive/1qpxrXC3YdJQzly3mRg-4ayYiOjC6rue3?usp=sharing

Reply:
Thanks for the feedback. Updated to use new pybind API.